emcc.py

```diff
@@ -309,7 +309,7 @@ def setup_environment_settings():
 
 
 def minify_whitespace():
-  return settings.OPT_LEVEL >= 2 and settings.DEBUG_LEVEL == 0
+  return settings.SHRINK_LEVEL >= 2 or (settings.OPT_LEVEL >= 2 and settings.DEBUG_LEVEL == 0)
```

Comment:
`-Oz` will set the opt level to 2. So this was already true, wasn't it, unless debugging is enabled?

Is the issue that you want a minified debug build for some reason?

Reply:
Ohh, yeah, I suppose I could/should turn off `-g` but I still think it would make sense if specifying `-Oz` enabled minification even with the default debug level - browsers can pretty-print js files for you but they can't enable source maps or whatever else `-g` does. Idk, if you think that it makes sense as is I'm fine with closing this.

Reply:
Alright, that makes sense. Thanks for your time :)